eclair-core/src/test/scala/fr/acinq/eclair/TestUtils.scala

```diff
@@ -27,4 +28,16 @@ object TestUtils {
     .props
     .get("buildDirectory") // this is defined if we run from maven
     .getOrElse(new File(sys.props("user.dir"), "target").getAbsolutePath) // otherwise we probably are in intellij, so we build it manually assuming that user.dir == path to the module
+
+  def availablePort: Int = synchronized {
```

Comment:
This change is to prepare parallelization, right?

Reply:
Yes, we now use a different set of ports each time we need to spawn bitcoind, in order to be able to run them concurrently.